### PR TITLE
Seperated periodic table functionality from elemental analysis widget

### LINKED ISSUE #32142: Seperate periodic table from elemental analysis gui
Periodic table functionality is in Elemental analysi GUI widget and should be put in seperate entity

### PR DESCRIPTION
**Pull request by tolu28-coder**
**Description of work.**
This pull request seperates all functionality relating to the periodic table in the elemental analysis gui into one entity.

**To test:**
1. As there was no changes to functionality, ensure tests pass and gui works as usual

Fixes #32142 . 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py [hunk 1]
```diff
@@ -94,18 +48,22 @@ def __init__(self, parent=None, window_flags=None):
...
-        self.peaks.electron.on_checkbox_unchecked(self.electrons_changed)
-        self.peaks.set_deselect_elements_slot(self.deselect_elements)
+        # plotting
+        self._plot_window = None
+        self._plotting = None
+
+        # setup periodic table and peaks
+        self.ptable_view = PeriodicTableView()
+        self.peaks_view = PeaksView()
+        self.ptable = PeriodicTableWidgetPresenter(self)
```
Comment (AnthonyLim23) [inline scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py:58]: why has the presenter not got a view?

Comment (tolu28-coder) [reply scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py:58]: Due to structure of ElementalAnalysisGUI widget i could not seperate all the widgets relating to the periodic table into one widget

**Comment by sf1919** [comment]
There is a problem with the GUI. It's no longer possible load any peaks data against a run. I did the following
1) Open Elemental Analysis GUI
2)  Ticked GE1
3) Loaded run ```2695``` - the plot loaded so the data seems to load fine
4) Clicked Cu on the periodic table
5) Ticked GE1
6) Got the following error message - there seems to be some missing functionality

Screenshot (14)


I will wait for this to be fixed before I continue testing.

File: scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py (hunk 1 above)
Comment (AnthonyLim23) [reply scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py:58]: That is the point of this piece of work. To make the periodic table a widget that obeys MVP.

**Comment by AnthonyLim23** [comment]
There is no one to work on this, so I am closing it.

**Pull request closed without merging**